pkg/roachprod/vm/vm.go

```diff
@@ -43,6 +43,10 @@ const (
 	ArchFIPS    = CPUArch("fips")
 	ArchUnknown = CPUArch("unknown")
 
+	// IncompatibleArchRequestError is returned when requested architecture
+	// is different than the architecture discovered from the machine-type
+	IncompatibleArchRequestError = `%s-machine-type %s arch (%s) incompatible with requested arch (%s)`
```

Comment:
It should be `const` [1].

[1] https://cockroachlabs.atlassian.net/wiki/spaces/CRDB/pages/181371303/Go+Golang+coding+guidelines#Format-Strings-outside-Printf

Reply:
It is :)

Reply:
Nice! Perhaps a reminder (to self) that it's best to review PRs in an IDE, where full context is more apparent.